status-im/status-react
# [Communities] Account selection: сan't see the entire text of the community rules

fixes #18747

[Figma](https://www.figma.com/file/h9wo4GipgZURbqqr1vShFN/Communities-for-Mobile?node-id=15331%3A285128&mode=dev)
[Figma design for scroll:](https://www.figma.com/file/h9wo4GipgZURbqqr1vShFN/Communities-for-Mobile?type=design&node-id=16409%3A80328&mode=design&t=74XCfD0sFyTstHQo-1)

### Summary
This minor pull request addresses an issue where users were unable to scroll to the end of the Community Rules if they were too long.

#### Platforms

- Android
- iOS

##### Functional

- public chats
- group chats


### Steps to test

1. Activate the Accounts Selection feature flag.
2. Input text of maximum length into the 'Dialog for new members' field on the desktop.
3. Invite a user and attempt to join this community on a mobile device.
4. Verify that the entire text is visible without truncation.


### Before and after screenshots comparison

Before:

https://private-user-images.githubusercontent.com/67952253/302988907-bec72cfb-0cbd-4edd-818f-df330bcada9b.mp4?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDczOTM5MDcsIm5iZiI6MTcwNzM5MzYwNywicGF0aCI6Ii82Nzk1MjI1My8zMDI5ODg5MDctYmVjNzJjZmItMGNiZC00ZWRkLTgxOGYtZGYzMzBiY2FkYTliLm1wND9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjA4VDEyMDAwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgxZWFhYjU1ZDdlNjJhNWU0NTg3NTYwM2MwYjNlYWY5MjlkM2QzNDA5ZDAxYTc2YTI3NzZmYzZjM2E3ZWJiNGMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.y4M5Nnd6YKK_x2f4DfL1lGIvRqJJZwtMUFG57DyPpAA

After:

https://github.com/status-im/status-mobile/assets/5999878/51031a50-c58b-4335-bfa9-16983d4efa55




status: ready
